reana_workflow_engine_serial/utils.py

```diff
@@ -228,28 +228,34 @@ def publish_workflow_failure(job_id,
                                       })
 
 
-def get_targeted_workflow_steps(workflow_json, target_step):
+def get_targeted_workflow_steps(workflow_json, target_step, from_step):
```

Comment:
What about giving `None` default values so that the client calls don't have to?  With the meaning: "If not specified, all workflow will be run".